otsd

```diff
@@ -104,8 +101,12 @@ calendar_path = os.path.expanduser(args.calendar_path)
 calendar = otsserver.calendar.Calendar(calendar_path)
 aggregator = otsserver.calendar.Aggregator(calendar)
 
+proxy = bitcoin.rpc.Proxy()
+info = proxy.getinfo()
+relayfee = float(info['relayfee'])
+
 stamper = otsserver.stamper.Stamper(calendar,
-                                    args.btc_min_relay_feerate / 1000 * bitcoin.core.COIN,
+                                    relayfee / 1000 * bitcoin.core.COIN,
```

Comment:
you are right, you could overpay... I am reverting this commit.
However you could overpay also if the relay fee is 1100 and the first try is 1000. You are gonna end paying 2000. Could be optimal to initialize the fee by asking the node every time the tx is initialized? (then doubling if the next block doesn't contain it)